frontend/src/metabase/dashboard/actions/cards.unit.spec.ts

```diff
@@ -362,7 +362,7 @@ async function runAddCardToDashboard({
   const nextState = store.getState();
 
   const tempDashCardId =
-    getDashboardById(nextState, dashId).dashcards.find(
+    getDashboardById(nextState, dashId)!.dashcards.find(
```

Comment:
forcing non-undefined since the selector signature has changed (it was incorrect before)